lib/puppet/provider/file_line/ssh.rb

```diff
@@ -37,7 +37,7 @@ def handle_create_with_match()
       raise Puppet::Error, "More than one line in file '#{resource[:path]}' matches pattern '#{resource[:match]}'"
     elsif match_count == 1
       Puppet.debug('Replacing config line')
-      transport.exec!("sed -i \"s/#{resource[:match]}.*/#{resource[:line]}/\" #{resource[:path]}")
+      transport.exec!("sed -i 's|#{resource[:match]}.*|#{resource[:line]}|' #{resource[:path]}")
```

Comment:
Shouldn't there be a similar change at https://vmware.webex.com/vmware/j.php?J=926032014 ?